packages/test-cases/cases/compose/sanity/output/module.ts

```diff
@@ -17,226 +17,268 @@ import {
   createInterfaceImplementedDefinition,
   createObjectRef,
   createEnvDefinition,
-  createAbi,
-  Abi,
+  WrapAbi,
 } from "@polywrap/schema-parse";
 
-export const abi: Abi = {
-  ...createAbi(),
+export const abi: WrapAbi = {
   envType: createEnvDefinition({
     properties: [
-      createScalarPropertyDefinition({ name: "foo", type: "String", required: true }),
```

Comment:
@cbrzn can you please disable auto formatting, it bloats the size of PRs and makes it difficult to identify where changes have occurred that are not just styling changes.